openedx/core/djangoapps/persistenttask/models.py

```diff
@@ -0,0 +1,61 @@
+"""
+Models to support persistent tasks.
+"""
+
+import json
+
+from django.db import models
+
+
+class FailedTask(models.Model):
```

Comment:
@nasthagiri @jibsheet I'm thinking the custom management could be handled by having the future management command/jenkins jobs pass the task name in.  I think using a separate table/abstract model will create more complexity and overhead, as each one will need its own task, and its own management commands that essentially do the same work.  

If we think separate tables are worth the overhead, we can implement them, but I want to make sure there's a tangible benefit.  I think indexing the table by task name might provide the main benefits we're looking for.